demo/config.yaml

```diff
@@ -20,3 +20,9 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
+
+    strelka:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+            candidate_small_indels_vcf: "results/manta-2.0/99-outputs/vcf/{seq_type}--{genome_build}/candidateSmallIndels/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf"
```

Comment:
I just realized that this file should be updated to:
```
results/manta-2.0/02-manta/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/results/variants/candidateSmallIndels.vcf.gz
```

Reply:
ya I think that's the issue, I'm rerunning it with the fixed one. Thank youu!

Reply:
I replaced this and now the issue is that strelka couldn't figure out the dependency, and so I got the MissingInput error. I'm thinking of using `"results/manta-2.0/99-outputs/vcf/{seq_type}--{genome_build}/candidateSmallIndels/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf"` as input but symlinking the original files still. Do you see a potential problem with this?

Reply:
I was under the impression that pointing to a file within an output directory would create that dependency link, but this and something I ran into yesterday has contradicted my impression. You could add the output directory as an input and the specific VCF file you want, but just don't use the directory in your command.

Reply:
Using the manta output directory as an input results in a childIO error when you rerun using the same samples. Do you know why this is?